README.md

```diff
@@ -23,7 +23,7 @@ Styleguide 2.1.3.
 a.button.star{
   ...
 }
-a.button.star.stars-given{
+a.button.star.star-given{
```

Comment:
`.star-given` is what is used in part of the SPEC and everywhere in the `test/` directory. Going to assume `.stars-given` is a typo.